src/Symfony/Component/Process/Pipes/WindowsPipes.php

```diff
@@ -55,6 +55,9 @@ public function __construct($input, $haveReadSupport)
                 foreach ($pipes as $pipe => $name) {
                     $file = sprintf('%s\\sf_proc_%02X.%s', $tmpDir, $i, $name);
 
+                    if (file_exists($file.'.lock')) {
```

Comment:
Since we cannot remove these files (see comment on the descriptor), this will lead to a huge amount of .lock files.
Instead, we should figure out how to `continue 2` when the fopen fails for access error reasons. An idea: if fopen fails, try to unlink. Then if unlink fails, `continue 2`.